search-app/docs/api.md

```diff
@@ -3227,6 +3227,8 @@ These parameters will match fields within a variable. They are case insensitive
   * options: pattern, ignore_case
 * native_id
   * options: pattern, ignore_case
+* variable_alias 
```

Comment:
I think the search parameter should be `alias` rather than `variable_alias`.

Reply:
Good point - this is on the /search/variables route so variable is redundant.

Reply:
Done.